src/java.base/share/classes/java/io/FileOutputStream.java

```diff
@@ -343,6 +343,7 @@ private native void writeBytes(byte[] b, int off, int len, boolean append)
      *
      * @param      b   {@inheritDoc}
      * @throws     IOException  {@inheritDoc}
+     * @throws     IndexOutOfBoundsException {@inheritDoc}
```

Comment:
When this IOOB can happen in `void write(byte[] b)` ?

Reply:
It can't. Thanks for catching this accident. Fixed in c3ee766e2a0e1bc4cc49c81a0eb3221b05c411b7.